packages/core/lib/networks.js

```diff
@@ -156,7 +156,7 @@ const Networks = {
       const fileContents = fs.readFileSync(filePath, "utf8");
       const body = JSON.parse(fileContents);
 
-      for (let installedNetworkId of body.networks) {
+      for (let installedNetworkId in body.networks) {
```

Comment:
`in` is used for iterating over object properties. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in

Reply:
I won't object strongly, but I think `for (let installedNetworkId of Object.keys(body.networks)) {` is probably cleaner syntax